WordPress/src/main/java/org/wordpress/android/reader/savedposts/resolver/ReaderSavedPostsResolver.kt

```diff
@@ -45,24 +41,9 @@ class ReaderSavedPostsResolver @Inject constructor(
 
         readerSavedPostsAnalyticsTracker.trackStart()
         appPrefsWrapper.saveIsFirstTryReaderSavedPostsJetpack(false)
-        val savedPostsCursor = getReaderSavedPostsCursor()
-        if (savedPostsCursor != null) {
-            val posts = queryResult.getValue<ReaderPostList>(savedPostsCursor) ?: ReaderPostList()
-
-            updateReaderSavedPosts(onSuccess, onFailure, posts)
-        } else {
-            readerSavedPostsAnalyticsTracker.trackFailed(ErrorType.QuerySavedPostsError)
-            onFailure()
-        }
-    }
-
-    private fun getReaderSavedPostsCursor(): Cursor? {
-        val savedPostsUriValue =
-                "content://${wordPressPublicData.currentPackageId()}.${ReaderSavedPostsProvider::class.simpleName}"
-        return contentResolverWrapper.queryUri(
-                contextProvider.getContext().contentResolver,
-                savedPostsUriValue
-        )
+        val (savedPosts) = localMigrationContentResolver.getDataForEntityType<ReaderPostsData>(ReaderPosts)
+        updateReaderSavedPosts(onSuccess, onFailure, savedPosts)
+//            readerSavedPostsAnalyticsTracker.trackFailed(ErrorType.QuerySavedPostsError)
```

Comment:
Should we uncomment or remove this line before merging?

Reply:
Handled w/ https://github.com/wordpress-mobile/WordPress-Android/pull/17424/commits/2663bd7fc6e49723d883ff28e69be2c12f86351b